notebooks/02_Fairing/README.md

```diff
@@ -7,7 +7,7 @@ I build a container image with awscli and docker support, Please use `seedjeffwa
 
 ### ECR permission
 
-Since fairing library will create ECR repository and upload repository to ECR. Grant `AmazonEC2ContainerRegistryFullAccess` to your node group role.  (We will use simplest policies instead later)
+Since fairing library will create ECR repository, upload repository to ECR, and put objects in S3 bucket. Grant `AmazonEC2ContainerRegistryFullAccess` and `AmazonS3FullAccess` to your node group role.  (We will use simplest policies instead later)
```

Comment:
Actually, we should figure out the granular permission instead of using `AmazonS3FullAccess`. This is minor and if you have time, it would be great to figure that out. I am also ok with full for short.

Reply:
Since I tried with `AmazonS3ReadOnly` and it doesn't work, for simplicity, I use `AmazonS3FullAccess`. I'll figure it out later.

Reply:
Sure.